asv_bench/benchmarks/benchmarks.py

```diff
@@ -376,38 +376,63 @@ def time_head(self, shape, head_count):
         execute(self.df.head(self.head_count))
 
 
-class TimeFillna:
-    param_names = ["self_type", "value_type", "shape", "limit", "inplace"]
+class TimeFillnaSeries:
```

Comment:
`Fillna` is divided into two classes. Is this for better display of the results? Should it be applied for the OmniSci set?

Reply:
Yes, to improve display of the results.

No, for OmniSci a simplified version of the benchmark is used, there are still fewer parameters there.